src/main/java/bio/terra/service/job/BaseStep.java

```diff
@@ -0,0 +1,13 @@
+package bio.terra.service.job;
+
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+
+public abstract class BaseStep implements Step {
```

Comment:
I wonder if there's a better name for this than `BaseStep`. `Base` doesn't really give me any context for what its doing. Something like `DefaultUndoStep` or `EmptyUndoStep` or along those lines would be more descriptive.

Reply:
I agree, this was a placeholder name I picked without thinking too much about it. I think `DefaultUndoStep` would work.